### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

File: remote_write/README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The repo tests the following remote write senders:
 - [InfluxData's Telegraf](https://github.com/influxdata/telegraf).
 - The [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
 - The [VictoriaMetrics Agent](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmagent), unless you're on [Mac OS X](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1042).
+- [Timber.io Vector](https://github.com/timberio/vector)
```
Comment (juliusv) [inline remote_write/README.md:36]: Nit: all other entries end with a period.

**Comment by juliusv** [comment]
DCO is failing...

**Comment by juliusv** [comment]
@RichiH The DCO check is still failing because the first commit doesn't have a signoff line. You may want to edit that (or squash) and force push.

**Comment by RichiH** [comment]
I clicked the thing to make DCO pass (web edit...) and it told me it worked. Squashed & force-pushed.

**Pull request merged**